src/login/Login.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Redirect } from "react-router-dom";
+import "../myspace/MySpace";
+
+
+export default function Login() {
+    const [username, setUsername] = useState(true);
+    const [password, setPassword] = useState(true);
+    const [responseMessage, setResponseMessage] = useState(null);
+    const [accessToken, setAccessToken] = useState(null);
+    const [accessExpiry, setAccessExpiry] = useState(null);
+    const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+    const handleSubmit = async e => {
+        e.preventDefault();
+        let userInput = {}
+
+        new FormData(e.target).forEach((value, key) => {
+            if (key === "username")
+                userInput[key] = value;
+            else if (key === "password")
+                userInput[key] = value;
+
+        });
+
+
+        const payload = {
+            username: userInput["username"],
+            password: userInput["password"],
+        };
+        const requestLogin = {
+            method: "POST",
+            headers: {
+                "Accept": "application/json",
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(payload)
+
+        };
+        let data = "";
+        let resStatus = 0;
+        fetch("http://127.0.0.1:5000/login", requestLogin)
+            .then(async response => {
+                resStatus = response.status
+                data = await response.json();
+            })
+            .then(res => {
+                switch (resStatus) {
+                    case 200:
+                        setAccessToken(data["access_token"]);
+                        setAccessExpiry(data["access_expiry"]);
+                        setIsAuthenticated(true);
+                        break
+                    default:
+                        setResponseMessage(data["message"]);
+                        break
+                }
+            })
+            .catch(error => {
+                setResponseMessage(data["message"]);
+            });
+    }
+
+    if (isAuthenticated === true) {
+        return <Redirect to="/my_space" />
+    }
+
+    return (
+        <div className="container">
+            <div className="row mb-5">
+                <div className="col-lg-12 text-center">
+                    <h1 className="mt-5">Login</h1>
+                </div>
+            </div>
+            <div className="row">
+                <div className="col-lg-12">
+                    <form className="login-form mx-auto" onSubmit={handleSubmit}>
+                        <form-group controlId="formUserame">
+                            <p className="input-control">
+                                <label htmlFor="username">Username or Email:</label>
```

Comment:
remove the or email. I might expect login via email too. It can be misleading at this point, we can look at it later

Reply:
done